src/util.cpp

```diff
@@ -150,7 +152,7 @@ void Util::parseArgs(int argc,
 #if USE_MMAL_ENCODER || USE_JETSON_ENCODER
         return std::string();
 #else
-        return "このデバイスは --use-native に対応していません。";
+        return "This machine does NOT support --use-native option.";
```

Comment:
cannot use --use-native option because your device does not have this feature.

Reply:
ditto